### DISCUSSION TITLE
Announcing AirGen - the next generation aerial robotics simulator

### DISCUSSION THREAD

**Discussion by saihv** [microsoft/AirSim#4889] [General]
## AirGen 

The original creators of AirSim are proud to present AirGen - the next generation of aerial robotics simulation that is focused on enabling aerial robot intelligence through large-scale data generation and evaluation for machine learning. 
Built on top of AirSim, AirGen adds several exciting new features and incorporates state-of-the-art technologies. 
AirGen is currently in open pre-release preview. 

## AirGen is free and unrestricted for academic users
Visit our web portal and sign up to access AirGen: https://portal.scaledfoundations.ai/. If you are an academic user, please sign up with an academic email address (.edu, .ac.* etc.) for unrestricted access.

## Features
![image](https://github.com/microsoft/AirSim/assets/2274262/897cf531-3bf9-4c39-b66f-33dcf2df2dc6)

**Unreal Engine 5.2 ready**: AirGen is built on top of Unreal Engine 5.2, enabling you to leverage the latest features in Unreal Engine such as Nanite, Lumen, and more. Furthermore, AirGen also has NVIDIA DLSS 3.0 support, enabling you to run AirGen at higher framerates while maintaining excellent visual fidelity.

**Geospecific Environment Support**: Dive into real-world terrains with AirGen's support for geospecific environments using Cesium and photorealistic tiles from Google Maps. Experience a seamless blend of the virtual and real world, delivering unparallel experience for your simulation needs. (Google Maps API key required)

**Advanced Path Planning**: With the integration of occupancy maps, signed distance fields, and path planning algorithms, AirGen enables generating collision-free trajectories at scale for safe navigation.

**Large-Scale Data Collection:** AirGen's support for large-scale data collection allows you to collect virtually infinite amounts of photorealistic multimodal data, unlocking the training of generalizable perception-action models.

**Digital Twin import**: AirGen supports importing meshes or scenes at runtime, enabling you to simulate real-world context within simulation.

AirGen plays a central role in the General Robot Intelligence Development (GRID) platform. We created GRID to help accelerate robotic development through the combination of AI foundation models and simulation. The powerful orchestration and interaction capabilities of large language models in GRID enable rapid prototyping of complex robotics tasks. 

Read more about our vision in our technical paper: https://scaledfoundations.ai/wp-content/uploads/GRID_paper.pdf

**Comment by EXPX3** [comment]
@saihv will there be ros2 support? can we use the existing ros wrapper from airsim? 

~~i was stuck using airsm and roswrapper (ros2), when i launch the airsim_node i get waiting for connection error, and not getting services like takeoff, land published. (this is solved it was a docker connection issue an dnot airsim related)~~
